rai/lib/numbers.cpp

```diff
@@ -311,6 +311,10 @@ bool rai::uint256_union::decode_dec (std::string const & text)
 		{
 			stream >> number_l;
 			*this = number_l;
+			if (!stream.eof ())
```

Comment:
This is to prevent parsing things like "1.1" as "1" until we implement it properly.